tests/integration/templates/nginx-dualstack.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
```

Comment:
i imagine we'll be writing a test that uses this? do we need this nginxdualstack image or does a simple `nginx` also work?

Reply:
It seems like you need to explicitly enable IPv6 in nginx. 
If you deploy the default nginx image and try to connect to it then it fails with:
```
root@k8s-dualstack:~# curl http://[fd98::89f]/
curl: (7) Failed to connect to fd98::89f port 80 after 0 ms: Connection refused
```

See also: https://ma.ttias.be/enable-ipv6-connectivity-for-nginx/

Reply:
I don't have strong opinions here - this was really just used for a first manual test - and I mainly added it so that @eaudetcobello can easily follow the steps I described in the documentation. Microk8s uses the same image and I just stole it from there.

Let's keep it like this for now. We may still update the image when writing the integration test.

Reply:
sounds good